zuul.d/nodesets.yaml

```diff
@@ -779,3 +779,47 @@
         nodes:
           - Trendmicro-DeepSecurity-20.0.344
           - ubuntu-bionic
+- nodeset:
+    name: Trendmicro-DeepSecurity-20.0.344-python27
+    nodes:
+      - name: fedora-35
+        label: fedora-35-1vcpu
+      - name: Trendmicro-DeepSecurity-20.0.344
+        label: Trendmicro-DeepSecurity-20.0.344
+    groups:
+      - name: appliance
```

Comment:
Please reuse the group organisation of `Trendmicro-DeepSecurity-20.0.344-python38`. You should have `appliance-ssh` and `aws` groups.